src/aat/resources/features/F-027/F-027.feature

```diff
@@ -0,0 +1,58 @@
+@F-027
+Feature: Fetch a case for display for Case Worker
+
+  Background: Load test data for the scenario
+    Given an appropriate test context as detailed in the test data source
+
+  @S-038
+  Scenario: must return status 200 along with the case-view object successfully
+    Given a user with [an active profile in CCD]
+    And a case that has just been created as in [Standard_Full_Case]
+    When a request is prepared with appropriate values
+    And the request [contains as input a valid case-reference that exists in CCD]
+    And it is submitted to call the [Fetch a case for display for Case Worker] operation of [CCD Data Store]
+    Then a positive response is received
+    And the response [has the 200 return code]
```

Comment:
Are we not using the case reference of the case object just created? If so, this fact not is reflected to scenario wording. Moreover, when we get the response, it should contain the details of the case just created, shouldn't it. If what I understand is correct, this scenario is not worded well to describe the relations with the case record just created.

Reply:
It is already there on the line number 10. If required I can remove line no 12 completely.

Reply:
Updated

Reply:
we should relate the case just created with the request being made